### PR TITLE
Fixes invalid status and ip error in volume info command

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@gmail.com>

**What this PR does / why we need it**:There was a bug in volume info command in which the status was incorrect . This PR fixes that bug .
screenshot from 2018-06-04 13-20-08

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
fixes: https://github.com/openebs/openebs/issues/1574

### CODE REVIEW

File: cmd/mayactl/app/command/volume_info.go [hunk 1]
```diff
@@ -23,6 +23,13 @@ var (
         `
 )
 
+//values keeps info of the values of a current address in replicaIPStatus map
+type values struct {
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volume_info.go]: values -- Value ?

File: cmd/mayactl/app/command/volume_info.go [hunk 2]
```diff
@@ -23,6 +23,13 @@ var (
...
+	i      int
```
Comment (prateekpandey14) [inline cmd/mayactl/app/command/volume_info.go]: i is needed here ? we already assigning below in range of `addressIPStrings`

Comment (ashishranjan738) [reply cmd/mayactl/app/command/volume_info.go]: yes i is needed here because it is used during formation of replica info

**Comment by NisantaSahoo** [comment]
Congratulations @ashishranjan738  for your PR getting merged! Thanks for taking part in OpenEBS Hackfest.
Fill up the doc form to get the Hackfest T-Shirt and Swags
https://docs.google.com/spreadsheets/d/1TZYIdERnD72YPeUsdS5hDfI6xgXXCax4-DHpXh-o6r0/edit?usp=sharing
Looking for some more awesome contributions from you and win more prizes

**Pull request merged**